joomla/joomla-cms
# [5.1] Rewrite com_associations in vanilla JS

### Summary of Changes

This PR addresses two items:

1. Two of the toolbar buttons ("Save Target" and "Copy Reference to Target") were missing an ID and therefore were not hidden as the JS couldn't find them.
2. Remove the jQuery dependency for the com_associations sidebyside JS

### Testing Instructions

1. Ensure you have a multilingual site setup.
2. Play around with com_associations and make sure it's all working as expected.
